lib/burnside/bridge.h

```diff
@@ -136,6 +136,9 @@ class BurnsideBridge {
   /// Cross the bridge from the Fortran parse-tree, etc. to FIR+OpenMP+MLIR
   void lower(const parser::Program &program, fir::NameUniquer &uniquer);
 
+  void dumpPreFIR(bool flag = true) { dumpPreFIR_ = flag; }
+  bool getDumpPreFIR() { return dumpPreFIR_; }
```

Comment:
We're following LLVM coding standards in the bridge code. None of this code uses Google's trailing "_".

Reply:
ok